index.html

```diff
@@ -25,9 +25,26 @@
 
 <!--<div id="overlayGIF"></div> overlayGIF details in main.css-->
     <div id="info-overlay">
+    
         <div id="text">
```

Comment:
Also I just noticed this, not a great id name :)
Does this get used for anything?